### DISCUSSION TITLE
New monorepo: fabric8/kubernetes-client

### DISCUSSION THREAD

**Discussion by rombert** [renovatebot/renovate#21459] [Suggest an Idea]
### Type of discussion.

I'm proposing an idea

### Tell us more.

Hi,

The https://github.com/fabric8io/kubernetes-client repository generates multiple Maven artifacts with the same version. Would it make sense to submit a new monorepo preset to renovate? I can give it a shot if the maintainers agree.

Here are the updates that Renovate would generate for one of my repositories without the grouping rule:

![image](https://user-images.githubusercontent.com/21237/231524644-73c6032b-bdfd-4183-b3b5-5b60f1b52820.png)

**Comment by secustor** [comment]
Yeah, make sense. Feel free to open a PR.
